### ISSUE TITLE
General Notes

### ISSUE THREAD

**Issue by asmaaamin** [GSG-G11/newsProject-TheTriples#29]
First of all, I'm super happy and honored to be your team lead, you did all a very great job <3
I have some small notes for you:

**For better web accessibility:**
- I noticed that some `<img>` have an empty alternative text, like this one for example: 
https://github.com/GSG-G11/newsProject-TheTriples/blob/84c3aabe5763f4f829a1a6c516da488db861949d/index.html#L14 

> if the user for some reason cannot view it (because of slow connection, an error in the src attribute, or if the user uses a screen reader). The value of the alt attribute should describe the image.

- I liked that you used some semantic HTML, but there are some lines that could also have a semantic HTML, like the `<main>` for the main part on the main page, and `<article>` for the news article on the details page. 
- For the details page, it would be better to give the page a descriptive title for it, like giving it the news title 
https://github.com/GSG-G11/newsProject-TheTriples/blob/84c3aabe5763f4f829a1a6c516da488db861949d/Details/details.html#L8

This is a very nice [article](https://medium.com/alistapart/writing-html-with-accessibility-in-mind-a62026493412) that has very useful tips for writing and HTML for better web accessibility.
and this is another [article](https://css-tricks.com/navigation-in-lists-to-be-or-not-to-be/) for writing a navbar for better accessibility too. 

**For the CSS:**
- Really really liked the `:root`, but why didn't you all benefit from it? some lines just have the color code, not the `var()`.

**For the code:**
- I liked that you did add some comments for some sections that it's related to the CSS, but it would be better if you also added a comment for every section that describes what's this section is for, I actually was a bit confused at reading the code and trying to know where this line belongs, so comments make the code more readable for anyone would read the code, and also for the team members if they want to review or edit anything.

**Finally**
I really liked how you did all worked together, loved your team spirit <3
keep it up <3
